content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -96,28 +96,15 @@ Once you install Istio and Kiali, deploy the [Bookinfo](/docs/examples/bookinfo/
         $ watch -n 1 curl -o /dev/null -s -w %{http_code} $GATEWAY_URL/productpage
         {{< /text >}}
 
-1.  To determine the Kiali URL, you use the same `GATEWAY_URL` as the Bookinfo application,
-    only with a different port.
+1.  Open the Kiali UI.
```

Comment:
How? This step is missing half the information the user need. I suggest adding it:

```suggestion
1. To open the Kiali UI, execute the following command in your Kubernetes environment:
```